test/unit/foundation.test.ts

```diff
@@ -131,4 +133,30 @@ describe('foundation', () => {
     it('does not render empty objects into its template', () =>
       expect(empty).dom.to.be.empty);
   });
+
+  describe('getDataSink', () => {
+    const doc = getDocument();
+    it('returns an array of ExtRef connected to the FCDA', () => {
+      // is checking that the ExtRef in the Privat element is NOT selected too
+      const fcda = doc.querySelector(
+        ':root > IED[name="IED2"] > AccessPoint > Server > LDevice[inst="CBSW"] > LN0 > DataSet > FCDA[daName="stVal"]'
+      )!;
+      expect(getDataSink(fcda)).to.have.lengthOf(1);
+    });
+
+    it('returns empty array if input not FCDA', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7203429)
```suggestion
    it("returns empty array if input not FCDA", () => {
```